test_data/rules/prefer_final_parameters.dart

```diff
@@ -68,11 +68,23 @@ void useGoodTypedClosureArgument() {
   _testingList.forEach((final int element) => print(element + 4)); // OK
 }
 
-void badMixed(String bad, final String good) { // LINT
+void badMixedLast(String bad, final String good) { // LINT
   print(bad);
   print(good);
 }
 
+void badMixedFirst(final String goodFirst, String badSecond) { // LINT
+  print(goodFirst);
+  print(badSecond);
+}
+
+// LINT [+1]
+void badMixedMiddle(String goodFirst, final String badSecond, String badThird) { // LINT
```

Comment:
@pq Is there a better way to identify multiple lints triggered on one line?

Reply:
Gross right? Not really, but at least it's possible...  Cheers to @a14n for that!

Reply:
You can be even more precise with something like:
```dart
// LINT [+2,column:length]
// LINT [+1,column:length]
void badMixedMiddle(String goodFirst, final String badSecond, String badThird) {
```